stacker_blueprints/rds/aurora/base.py

```diff
@@ -248,39 +248,41 @@ def create_parameter_group(self):
             )
 
     def create_cluster(self):
-        t = self.template
+        self.template.add_resource(self.cluster)
+
+    @property
+    def cluster(self):
         variables = self.get_variables()
         parameter_group = NoValue
         if variables["ClusterParameters"]:
             parameter_group = Ref(PARAMETER_GROUP)
+        else:
+            parameter_group = "default.%s" % variables["DBFamily"]
```

Comment:
Doesn't this happen by default when `NoValue` is given?

Reply:
It did, but for the `aurora-mysql` Engine, even when specifying `EngineVersion=5.7.12`, we're getting this error from CloudFormation:
```
The Parameter Group default.aurora5.6 with DBParameterGroupFamily oscar5.6 cannot be used for this instance. Please use a Parameter Group with DBParameterGroupFamily aurora-mysql5.7 (Service: AmazonRDS; Status Code: 400; Error Code: InvalidParameterCombination; Request ID: 6ee11fd2-15da-4854-97af-772ec8977885)
```
so it seems the default for that version is skewed. This may be fixed in the future, but given that the default parameter group for each family has a documented name of `default.<name>`, I think it makes sense to provide it.